packages/unchained-client/src/types.ts

```diff
@@ -72,6 +74,8 @@ export interface BaseTxMetadata {
 
 export interface StandardTxMetadata extends BaseTxMetadata {
   parser: `${TxParser}`
+  value?: string
+  assetId?: AssetId
```

Comment:
we should not be polluting the standard tx metadata with any parser specific data
```suggestion
```